lib/TestSuiteRunner.ts

```diff
@@ -149,11 +168,11 @@ ${LogSymbols.error} ${result.test.name}
 
     // Describe application
     const app = properties.applicationUri;
-    quads.push(quad(app, p.rdf  + 'type', p.earl + 'Software'));
```

Comment:
My alignments 😱

Reply:
We should really look into fixing this vscode / webstorm formatting discrepancy :open_mouth:

Reply:
It looks like there are no tslint rules for this atm.

However, there are rules for this in eslint, and since tslint and eslint will be merged (https://github.com/palantir/tslint/issues/4534), we will be able to use this in the future.

Reply:
Formatting rules seem to actually get removed (or there are maintainers hinting at it at least) according to that issue. They imply dedicated formatters should be used like Prettier.

Reply:
I don't fully agree with them.

Reply:
Oh, that's interesting.

I can actually understand the reasoning behind that choice, as code formatting is not really linting.

In any case, that's something for the future when we migrate to this new tooling. But thanks for bringing it to my attention!